databases/lmdb/Portfile

```diff
@@ -9,14 +9,16 @@ PortGroup           makefile 1.0
 # https://trac.macports.org/ticket/62272
 legacysupport.newest_darwin_requires_legacy 9
 
-github.setup        LMDB lmdb 0.9.31 LMDB_
```

Comment:
It might make sense to shift to using the Gitlab project, and thus using the `gitlab` portgroup.

https://git.openldap.org/openldap/openldap/-/tree/LMDB_0.9.33?ref_type=tags

So it might look something like:
```
gitlab.setup openldap openldap 0.9.33 LMDB_
name lmdb
```

In this way, you shouldn't have to set `master_sites` and `distname`.

Reply:
Great suggestion for improvement! I had been thinking something similar was possible but didn't go the extra mile to test such things. Thanks for the insights!